cv_next/app/feed/components/inputbar.tsx

```diff
@@ -0,0 +1,48 @@
+export const InputBox = ({
```

Comment:
If you want, you can wrap these two components (if they have similar semantic meaning) in a wrapper component with a `variant` value to choose between the two

Reply:
I prefer 2 different components, it is more readable, maintainable, and is more clean.
there is principle but i don't remember its name